docs/use-map.mdx

```diff
@@ -0,0 +1,55 @@
+---
+name: useMap
+route: /map
+---
+
+import { Playground, Props } from "docz"
+import { useMap } from "../src"
+
+# useMap
+
+```js
+function App() {
+  const map = useMap();
+  // just like useState, you can pass
+  // a default value (e.g. new Map([["key", "value"]]))
+
+  const toggleExistence = () => {
+    if (map.has('key')) {
+      map.delete('key');
+    } else {
+      map.set('key', 'value');
+    }
+  };
+
+  return (
+    <>
+      <p>Map has value for 'key': {map.has('key').toString()}</p>
+      <button onClick={toggleExistence}>Toggle 'Key'</button>
+    </>
+  );
+}
+```
+
+## Display
```

Comment:
Change this section's name to "Example"

Reply:
Please add one more `#` so we don't get this submenu here:

![image](https://user-images.githubusercontent.com/26530524/68001037-1eb89d80-fc41-11e9-8fc1-1cd6a9c6be25.png)

Reply:
Done